docs/how-to/download-new-upstream-version.rst

```diff
@@ -1,9 +1,139 @@
 ..  _download-new-upstream-version:
 
-===============================
-Download a new Upstream Version
-===============================
+Download a new :term:`Upstream` version
+=======================================
 
 .. caution::
 
     Work in progress!
+
+Once in a while you need to download a new :term:`Upstream` release or check if a newer
```

Comment:
```suggestion
Once in a while you may need to download a new :term:`Upstream` release or check if a newer
```

Reply:
I would also, as a user (particularly a new user) ask "why/under what circumstances?"

Reply:
I think using one example (max two) is fine, and I'd say probably whichever you think is most relevant to someone who might be approaching this task for the first time. We don't want to turn this into an explanation page, but still, for a newer developer it might not be obvious why this task is useful.